README.md

```diff
@@ -146,6 +147,15 @@ thickness at each scale (`false`).
 
 __NOTE:__ The canvas is automatically updated when this value is changed.
 
+### SnappyContext2D.scaleDashLine
+
+    ctx.scaleDashLine = <Boolean>
+
+Determines if the dash line is scaled (`true`, default) or if it keeps the same
```

Comment:
Determines if the width of the dashes of dashed lines are scaled (`true`,
default) or if they keep the same width at each scale (`false`).